README.md

```diff
@@ -152,12 +186,29 @@ CLOUDANT_PASSWORD=<password>
 **Note**: We recommend using [IAM](#iam-authentication) for Cloudant and
 [Session](#session-cookie-authentication) for CouchDB authentication.
 
+#### Authenticate with external configuration or programmatically
+
+To use an external configuration file, the
+[Cloudant API docs](https://cloud.ibm.com/apidocs/cloudant?code=python#authentication-with-external-configuration),
+or the
+[general SDK usage information](https://github.com/IBM/ibm-cloud-sdk-common#using-external-configuration)
+will guide you.
+
+To learn more about how to use programmatic authentication, see the related documentation in the
```

Comment:
I would add here an additional section. `Authenticate programmatically`

Reply:
Done in 2fd6873.

Reply:
I prefer this section to be `Authenticate programmatically` as Vera suggested and it would be consistent with the other sections.

Reply:
It is in eec751f.